guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,478 @@
 {
```

Comment:
<ol style="user-select: auto;"><li style="user-select: auto;">typo in "suppport"</li><li style="user-select: auto;">Adds <code style="user-select: auto;">geometry_column</code> parameter to <code style="user-select: auto;"><a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html#arcgis.features.GeoAccessor.from_df" target="_blank" style="user-select: auto;">Spatially Enabled DataFrame.from_df()</a></code> method -&gt; pd.DataFrame.spatial.from_df()</li><li style="user-select: auto;"><br style="user-select: auto;"></li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Thanks. Made those edits.</p>